### ISSUE TITLE
Wii U Menu - User Mii editer/changer

### ISSUE THREAD

**Issue by mitch03** [cemu-project/Cemu#1015]
I would love for cemu to be able to change and edit the user mii, i dumped all my account data and would love to be able to change my user mii inside cemu Wii U Menu

**Comment by Haxorzz** [comment]
Just use the Mii channel?

**Comment by BLiNXthetimesweeperGOD** [comment]
You can't (at least not as far as I know), as you need to do it from the user settings but the emulated system gets softlocked (doesn't crash, just stops letting you do stuff) when it tries to switch apps.

**Comment by TomTurbine** [comment]
> You can't (at least not as far as I know), as you need to do it from the user settings but the emulated system gets softlocked (doesn't crash, just stops letting you do stuff) when it tries to switch apps. Trying to edit a user Mii from the Mii Maker just leads to it saying "You can edit or change your user Mii from User Settings. You cannot edit or delete user Mii characters in Mii Maker."

If you dump your entire system, account and all, it allows you to use your Mii Maker.

If can access the Mii Maker but the first one is completely garbled. But any extra you make from that point on works just fine.

But do hope they eventually include a Mii Maker applet like the Switch emulators include.

**Comment by rrrcabral** [comment]
i want delete that ugly default mii too

**Comment by Peter11117** [comment]
hello, can you create a mii and account please for me to cemu in your wii u please? I can share you my mii desing pleasee my friend

**Comment by reecespieces** [comment]
> You can't (at least not as far as I know), as you need to do it from the user settings but the emulated system gets softlocked (doesn't crash, just stops letting you do stuff) when it tries to switch apps. Trying to edit a user Mii from the Mii Maker just leads to it saying "You can edit or change your user Mii from User Settings. You cannot edit or delete user Mii characters in Mii Maker."

This is correct. The issue is that Cemu cannot switch apps, so the Settings app cannot open the Mii Maker app, which is the **only** way to do it on a real WiiU: https://www.youtube.com/watch?v=_K_ue-99raI

Until the Settings App can launch the Mii Maker app, the **only** way to change the **designated user Mii assigned to the Main account**, is to change it on a real WiiU and them dump your account.dat file again. 

Cemu can launch the Mii Maker from the home menu, but not from the Settings app. Because of this, Cemu can only be used to edit all the other Miis. it cannot be used to edit the **designated/main user** Mii.

**Comment by speedyspikes** [comment]
I just found this resource for changing the user's Mii in Cemu using Mii Creator. 
[https://github.com/L-Dev31/How-To-Replace-Cemu-s-Default-Mii?tab=readme-ov-file](https://github.com/L-Dev31/How-To-Replace-Cemu-s-Default-Mii?tab=readme-ov-file)
Now I'm trying to figure out how to add other Miis from Mii Creator to the Mii database in Cemu.

**Comment by cleementy** [comment]
There is this tool where it can generate an account based of your Mii on your Pretendo account.
You can customise this user and you should give it your pretendo account and password. It is not a scam and works. 
Lastly, this may be the only way for users that do not have a Wii U system.
The link: https://github.com/TheCraZyDuDee/Pretendo-Account-Generator
 
![Image](https://github.com/user-attachments/assets/658b6253-9f5b-4c35-9846-2fe5b0f49b05)

**Comment by anahbmp** [comment]
Follow the instructions on this page, it worked for me! https://github.com/L-Dev31/How-To-Replace-Cemu-s-Default-Mii?tab=readme-ov-file
